VogtPayroll3/Display.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll3
+{
+    class Display
+    {
+
+        public Dictionary<int, Employee> PrintMenu()
+        {
+            Employee emp = new Employee("holder", "holder", 1, 1, 1.0m);
```

Comment:
Why would you create an employee like this? If you have to create an employee "holder", that is a code smell that your design is off. This goes along with my last comment though -- the employee should NOT be creating another employee.